quora-api/src/main/java/com/upgrad/quora/api/controller/AdminController.java

```diff
@@ -1,4 +1,34 @@
 package com.upgrad.quora.api.controller;
 
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+import com.upgrad.quora.api.model.UserDeleteResponse;
+import com.upgrad.quora.service.business.UserBusinessService;
+import com.upgrad.quora.service.exception.AuthorizationFailedException;
+import com.upgrad.quora.service.exception.UserNotFoundException;
+
+@RestController
+@RequestMapping("/")
```

Comment:
replace this with @RequestMapping("/api/v1/") since it is mentioned in json file